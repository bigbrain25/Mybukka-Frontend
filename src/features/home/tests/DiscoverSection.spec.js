import React from 'react';

import DiscoverSection
  from '../components/DiscoverSection';

describe('Area to explore component', () => {
  const wrapper = shallow(<DiscoverSection />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
