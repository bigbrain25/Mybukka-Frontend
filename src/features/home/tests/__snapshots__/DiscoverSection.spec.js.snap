// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Area to explore component renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DiscoverSection />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="row"
      >
        <div
          className="col-lg-6 col-md-6 col-sm-12 col-xs-12"
        >
          <LargeText />
          <div
            className="d-none d-md-block"
          >
            <SmallText />
            <ActionButton
              handleClick={[Function]}
            />
          </div>
        </div>
        <div
          className="col-lg-6 col-md-6 col-sm-12 col-xs-12"
        >
          <SideImage />
          <div
            className="d-md-none"
          >
            <SmallText />
            <ActionButton
              handleClick={[Function]}
            />
          </div>
        </div>
      </div>,
      "className": "container discover-section",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="col-lg-6 col-md-6 col-sm-12 col-xs-12"
          >
            <LargeText />
            <div
              className="d-none d-md-block"
            >
              <SmallText />
              <ActionButton
                handleClick={[Function]}
              />
            </div>
          </div>,
          <div
            className="col-lg-6 col-md-6 col-sm-12 col-xs-12"
          >
            <SideImage />
            <div
              className="d-md-none"
            >
              <SmallText />
              <ActionButton
                handleClick={[Function]}
              />
            </div>
          </div>,
        ],
        "className": "row",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <LargeText />,
              <div
                className="d-none d-md-block"
              >
                <SmallText />
                <ActionButton
                  handleClick={[Function]}
                />
              </div>,
            ],
            "className": "col-lg-6 col-md-6 col-sm-12 col-xs-12",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <SmallText />,
                  <ActionButton
                    handleClick={[Function]}
                  />,
                ],
                "className": "d-none d-md-block",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "handleClick": [Function],
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <SideImage />,
              <div
                className="d-md-none"
              >
                <SmallText />
                <ActionButton
                  handleClick={[Function]}
                />
              </div>,
            ],
            "className": "col-lg-6 col-md-6 col-sm-12 col-xs-12",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <SmallText />,
                  <ActionButton
                    handleClick={[Function]}
                  />,
                ],
                "className": "d-md-none",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "handleClick": [Function],
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="row"
        >
          <div
            className="col-lg-6 col-md-6 col-sm-12 col-xs-12"
          >
            <LargeText />
            <div
              className="d-none d-md-block"
            >
              <SmallText />
              <ActionButton
                handleClick={[Function]}
              />
            </div>
          </div>
          <div
            className="col-lg-6 col-md-6 col-sm-12 col-xs-12"
          >
            <SideImage />
            <div
              className="d-md-none"
            >
              <SmallText />
              <ActionButton
                handleClick={[Function]}
              />
            </div>
          </div>
        </div>,
        "className": "container discover-section",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="col-lg-6 col-md-6 col-sm-12 col-xs-12"
            >
              <LargeText />
              <div
                className="d-none d-md-block"
              >
                <SmallText />
                <ActionButton
                  handleClick={[Function]}
                />
              </div>
            </div>,
            <div
              className="col-lg-6 col-md-6 col-sm-12 col-xs-12"
            >
              <SideImage />
              <div
                className="d-md-none"
              >
                <SmallText />
                <ActionButton
                  handleClick={[Function]}
                />
              </div>
            </div>,
          ],
          "className": "row",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <LargeText />,
                <div
                  className="d-none d-md-block"
                >
                  <SmallText />
                  <ActionButton
                    handleClick={[Function]}
                  />
                </div>,
              ],
              "className": "col-lg-6 col-md-6 col-sm-12 col-xs-12",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <SmallText />,
                    <ActionButton
                      handleClick={[Function]}
                    />,
                  ],
                  "className": "d-none d-md-block",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {},
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "handleClick": [Function],
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                ],
                "type": "div",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <SideImage />,
                <div
                  className="d-md-none"
                >
                  <SmallText />
                  <ActionButton
                    handleClick={[Function]}
                  />
                </div>,
              ],
              "className": "col-lg-6 col-md-6 col-sm-12 col-xs-12",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <SmallText />,
                    <ActionButton
                      handleClick={[Function]}
                    />,
                  ],
                  "className": "d-md-none",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {},
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "handleClick": [Function],
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                ],
                "type": "div",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
